patterns/services-three-columns.php

```diff
@@ -27,8 +27,8 @@
 			</figure>
 			<!-- /wp:image -->
 
-			<!-- wp:heading {"level":4} -->
-			<h4 class="wp-block-heading">Collect</h4>
+			<!-- wp:heading {"level":3} -->
+			<h3 class="wp-block-heading">Collect</h4>
```

Comment:
This changes the level but does not correct the font size, so the design will not match
Since the heading sizes are not available, I think this is acceptable for now.

Reply:
Previous size at the top, new size below:
![image](https://github.com/user-attachments/assets/f2c7a53b-04a2-4eba-8ac6-f684de454530)